rv1:  # Name of your Node
  ros__parameters:
    init_pos_x: 396.2
    init_pos_y: -250.0
    init_pos_z: 0.0
    init_rot_x: 0.0
    init_rot_y: 0.0
    init_rot_z: 0.7072
    init_rot_w: 0.7072
    goal_pos_x: 396.413
    goal_pos_y: -24.0
    goal_pos_z: 0.0
    goal_rot_x: 0.0
    goal_rot_y: 0.0
    goal_rot_z: 0.7072
    goal_rot_w: 0.7072
    emergency_break_distance: 100.0

rv2:  # Name of your Node
  ros__parameters:
    init_pos_x: 396.2
    init_pos_y: -316.0
    init_pos_z: 0.0
    init_rot_x: 0.0
    init_rot_y: 0.0
    init_rot_z: 0.7072
    init_rot_w: 0.7072
    goal_pos_x: 396.413
    goal_pos_y: -24.0
    goal_pos_z: 0.0
    goal_rot_x: 0.0
    goal_rot_y: 0.0
    goal_rot_z: 0.7072
    goal_rot_w: 0.7072

hv:  # Name of your Node
  ros__parameters:
    init_pos_x: 396.2
    init_pos_y: -316.0
    init_pos_z: 0.0
    init_rot_x: 0.0
    init_rot_y: 0.0
    init_rot_z: 0.7072
    init_rot_w: 0.7072
    goal_pos_x: 396.2
    goal_pos_y: -20.0
    goal_pos_z: 0.0
    goal_rot_x: 0.0
    goal_rot_y: 0.0
    goal_rot_z: 0.7072
    goal_rot_w: 0.7072